### PR TITLE
Added default instrumentation config for fallback

### PR DESCRIPTION
**Pull request by praveenbalu**
Added default instrumentation config for fallback mechanism

### CODE REVIEW

File: buildpack/runtime.py [hunk 1]
```diff
@@ -75,6 +75,22 @@ def stage(buildpack_dir, build_path, cache_path):
         os.path.join(build_path, ".local", "m2ee.yaml"),
     )
 
+    util.mkdir_p(os.path.join(build_path, ".local", "mx-agent"))
```
Comment (markbastiaans) [inline buildpack/runtime.py:78]: Please don't add this to `runtime.py`, but to the staging phase in `mx-java-agent.py`.

**Review by markbastiaans — CHANGES_REQUESTED** [review]
In general, LGTM, but please add the copying of the default config to the staging phase in `mx-java-agent.py`.

File: buildpack/runtime.py (hunk 1 above)
Comment (praveenbalu) [reply buildpack/runtime.py]: Done

**Comment by carbone-kid** [comment]
Looks good to me.

**Review by carbone-kid — APPROVED** [review]

**Pull request merged**